packages/extension-vscode/src/quickfix-provider.ts

```diff
@@ -0,0 +1,89 @@
+import { CodeAction, CodeActionKind, CodeActionParams, Command, Diagnostic, TextDocuments } from 'vscode-languageserver';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+import { getFeatureNameFromDiagnostic } from './utils/problems';
+
+export class QuickFixActionProvider {
+    private documents: TextDocuments<TextDocument>;
+    private sourceName: string;
+
+    public constructor(documents: TextDocuments<TextDocument>, sourceName: string) {
+        this.documents = documents;
+        this.sourceName = sourceName;
+    }
+
+    private createIgnoreFeatureAction(hintName: string, diagnostic: Diagnostic): CodeAction {
+        const command = 'vscode-webhint/ignore-feature-project';
+        const featureName = getFeatureNameFromDiagnostic(diagnostic);
+
+        if (!featureName) {
+            throw new Error('Unable to determine which HTML/CSS feature to ignore');
+        }
+
+        const action = CodeAction.create(
+            `Ignore '${featureName}' in this project`,
```

Comment:
Looking at this again with the new scoping, this might be even better to call out as compatibility related.
```suggestion
            `Ignore '${featureName}' compatibility in this project`,
```